src/pages/search.jsx

```diff
@@ -112,7 +112,9 @@ function SearchPage({
   // If we're using the default filters, use the products from the Gatsby data layer.
   // Otherwise, use the data from search.
   const isDefault = !data
-  const productList = (isDefault ? products.edges : data?.products?.edges) ?? []
+  const productList = React.useMemo(() => {
```

Comment:
Let's keep unrelated changes out of scope here. Was this causing issues with one of the updated packages?

Reply:
no, this was just a performance improvement - the console had the suggestion to memoize this and it was a pretty easy change so i did it

Reply:
reverting now